lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,104 @@
-
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
-  });
+  it('renders a threat indicator if policyThreatLevel prop is provided', async function() {
+    const { container } = quickRender({ policyThreatLevel: 2 }),
+        user = userEvent.setup(),
+        threatIndicator = container.querySelector('svg')!;
 
-  it('sets the modifier class by converting the policyThreatLevel if it is provided', function() {
-    expect(getShallowComponent({ policyThreatLevel: 0 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--none');
-    expect(getShallowComponent({ policyThreatLevel: 1 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--low');
-    expect(getShallowComponent({ policyThreatLevel: 3 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--moderate');
-    expect(getShallowComponent({ policyThreatLevel: 5 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--severe');
-    expect(getShallowComponent({ policyThreatLevel: 9 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--critical');
-    expect(getShallowComponent({ policyThreatLevel: 9 }).find('.nx-threat-indicator'))
-        .not.toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Moderate');
   });
 
-  it('sets the modifier class using the threatLevelCategory if both props are provided', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
-    const icon = component.find(NxFontAwesomeIcon);
+  it('takes precedence of threatLevelCategory if both props are provided', async function() {
+    const { container } = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        user = userEvent.setup(),
+        threatIndicator = container.querySelector('svg')!;
 
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--critical');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Low');
   });
 
   it('adds aria attrs to help the icon show up for screen readers', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
-
-    expect(component.find(NxFontAwesomeIcon)).toHaveProp('aria-label', 'threat level low');
-    expect(component.find(NxFontAwesomeIcon)).not.toHaveProp('aria-hidden');
-  });
+    const { container } = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        threatIndicator = container.querySelector('svg')!;
```

Comment:
Rather than querying for the svg everywhere, just use the top-level element of the render i.e. `container.firstElementChild`.  And of course, we have `rtlRenderElement`/`renderEl` to make that more convenient.

Reply:
> Rather than querying for the svg everywhere, just use the top-level element of the render i.e. `container.firstElementChild`. And of course, we have `rtlRenderElement`/`renderEl` to make that more convenient.

Changed to query by roles here: 4c116c4eab2f01f2c1f3089e42a7d0c415cc10c2